src/foam/u2/wizard/axiom/AnalyticWizardAction.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * @license
+ * Copyright 2023 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+  
+ foam.CLASS({
+    package: 'foam.u2.wizard.axiom',
+    name: 'AnalyticWizardAction',
+    extends: 'foam.u2.wizard.axiom.WizardAction',
+  
+    documentation: 'WizardAction with analytic events',
+  
+    imports: [
+      'analyticsAgent?'
+    ],
+
+    properties: [
+			{
+				name: 'message',
+				type: 'String'
+			},
+    	{
+    	  name: 'code',
+    	  value: function (slot, X) {
+    	    analyticsAgent?.pub('event', {
```

Comment:
Can this just be a feature on WizardAction? Think it works just as well and is way less code